freqtrade/freqai/prediction_models/ReinforcementLearner.py

```diff
@@ -100,7 +100,6 @@ def calculate_reward(self, action: int) -> float:
             """
             # first, penalize if the action is not valid
             if not self._is_valid(action):
-                self.custom_info["Invalid"] += 1
```

Comment:
Why are you removing this one?

Reply:
I originally added the "invalid", "hold" and "unknown" values when I tested the maskable ppo. I think logging actions that are not explicitly specified in model actions can be confusing for users and should not be present by default. But I can return them back if you think otherwise.

Reply:
If a user wants to understand how the agent navigated the reward, it makes sense to see how many times it made an invalid choice. Having stats on how many times the agent decided to hold us also important no? I don’t see how this would confuse a user.

 I also don’t quite understand the connection to maskable ppo.

Reply:
Please look at the new commit. I propose to remove all metrics from the reward function, and move the new built-in action counters to step(). In my maskable ppo example, I don't want to track the invalid and unknown actions in the tensorboard because they are always zero. But if we define a counter on reset, it will show up even if I remove the metric from the reward function. So I think it's better to display the bare minimum and leave the user the freedom to add whatever they want